cmds/root.go

```diff
@@ -16,8 +16,11 @@ var build string
 var arch string = fmt.Sprintf("%s-%s", runtime.GOOS, runtime.GOARCH)
 var noPrefix bool
 var force bool
+var ownershipFilename string
 
 func init() {
+	ownershipFilename = os.Getenv("TESTTRACK_OWNERSHIP_FILE")
```

Comment:
Should this work with the dotenv behavior below? (url and app name)

Reply:
Was a bit confused on this block of code in relation to `getAppName` below.

I believe I want to set `ownershipFilename` at the root level to `TESTTRACK_OWNERSHIP_FILE` either from raw ENV variables or from the `.env.local` file. 

Re-reading this now, seems like this will set `TESTTRACK_OWNERSHIP_FILE` from the `ENV` but I don't think it'll snag it from the `.env.local`. 

Any pointers on what I need to change to fix that?

Reply:
I was noodling around with this a little and came up with this commit on top of your branch (I can't push to your fork):

https://github.com/smudge/testtrack-cli/commit/11e50788f513f4b666b98840a2558c14bc2537bd

The idea here is that we load dotenv by default (which [their docs suggest doing](https://github.com/joho/godotenv#usage)) and then I pushed the actual `LookupEnv` call down into `validations`, so that the caller doesn't need to know that the `ValidateOwnerName` call relies on the presence of a filename that must be passed in.

Feel free to use any or none of that, but the main thing is that the dotenv stuff can be inverted to load first, rather than checking for existing ENV vars and then conditionally loading.

Reply:
Nice - Incorporated that.